src/restangular.js

```diff
@@ -1279,23 +1279,28 @@ module.provider('Restangular', function() {
       }
 
       function toService(route, parent) {
-        var knownCollectionMethods = _.values(config.restangularFields);
-        var serv = {};
-        var collection = (parent || service).all(route);
-        serv.one = _.bind(one, (parent || service), parent, route);
-        serv.post = _.bind(collection.post, collection);
-        serv.getList = _.bind(collection.getList, collection);
-
-        for (var prop in collection) {
-          if (collection.hasOwnProperty(prop) && _.isFunction(collection[prop]) && !_.contains(knownCollectionMethods, prop)) {
-            serv[prop] = _.bind(collection[prop], collection);
+          var knownCollectionMethods = _.values(config.restangularFields);
+          var serv = {};
+
+          var allMethod = parent ? config.restangularFields.all : "all";
+          var postMethod = config.restangularFields.post || "post";
+          var getListMethod = config.restangularFields.getList || "getList";
+
+          var collection = (parent || service)[allMethod](route);
+
+          serv.one = _.bind(one, (parent || service), parent, route);
+          serv.post = _.bind(collection[postMethod], collection);
+          serv.getList = _.bind(collection[getListMethod], collection);
+
+          for (var prop in collection) {
+            if (collection.hasOwnProperty(prop) && _.isFunction(collection[prop]) && !_.contains(knownCollectionMethods, prop)) {
+              serv[prop] = _.bind(collection[prop], collection);
+            }
```

Comment:
The indentation has changed here, making it seem that more has been changed than actually has. Fix the indentation to show the actual changes.